devenv/dev-dashboards/gen.libsonnet

```diff
@@ -0,0 +1,694 @@
+local grafana = import "grizzly/grafana.libsonnet";
```

Comment:
Need one of dem big honking `THIS FILE IS GENERATED, EDITING IS FUTILE` comment header blocks :)

Reply:
good call, added with [a4ec3b8](https://github.com/grafana/grafana/pull/57928/commits/a4ec3b8d57f6de7f2b9c20139276d2a2023bb8c5)